src/recording/recordingmanager.cpp

```diff
@@ -213,6 +249,28 @@ long RecordingManager::getFileSplitSize()
          return SIZE_2GB;
      else if(fileSizeStr == SPLIT_4096MB)
          return SIZE_4GB;
+     else if(fileSizeStr == SPLIT_60MIN)
+         return SIZE_4GB; //Ignore size limit. use time limit
+     else if(fileSizeStr == SPLIT_74MIN)
+         return SIZE_4GB; //Ignore size limit. use time limit
+     else if(fileSizeStr == SPLIT_80MIN)
+         return SIZE_4GB; //Ignore size limit. use time limit
+     else if(fileSizeStr == SPLIT_120MIN)
+         return SIZE_4GB; //Ignore size limit. use time limit
      else
          return SIZE_650MB;
 }
+long RecordingManager::getFileSplitTime()
+{
+    QString fileSizeStr = m_pConfig->getValueString(ConfigKey(RECORDING_PREF_KEY, "FileSize"));
+    if(fileSizeStr == SPLIT_60MIN)
+        return 60*60;
+    else if(fileSizeStr == SPLIT_74MIN)
+        return 74*60;
+    else if(fileSizeStr == SPLIT_80MIN)
+        return 80*60;
+    else if(fileSizeStr == SPLIT_120MIN)
+        return 120*60;
+    else // Do not limit by time for the rest.
+        return 999999999;
```

Comment:
Yes and no. Maybe for the method, but not for the comparison. If I were to use -1 (or 0, which is what I initially wanted to use), then the if ( x > split_time ) would need to be if ( split_time != -1 && x > split_time).
Sure, this way it is not really unlimited, and the method is only called once each second, but I would prefer the simpler if.